rfcs/cross_context_messaging.md

```diff
@@ -0,0 +1,122 @@
+# RFC 90: `testdriver` Add a cross-context messaging primitive
+
+## Summary
+
+Add `send`/`poll`/`recv` methods to testdriver that can be used to
+send messages to other browsing contexts, and receive messages for
+the current browsing context.
+
+## Details
+
+In tests that involve multiple origins, or multiple browsing context
+groups, there's often a need to make asserts about the behaviour of
+browsing contexts other than the test context. This problem has been
+solved by the COOP tests with
+[dispatcher.js](https://github.com/web-platform-tests/wpt/blob/7b0ebaccc62b566a1965396e5be7bb2bc06f841f/html/cross-origin-opener-policy/resources/dispatcher.js). This
+provides functions `send(uuid, message)` and `receive(uuid,
```

Comment:
Where does the UUID come from?
You said we can't generate them at will, but there will be instead one per context predefined.
Do we have an UUID for every type of context? (document, sharedworker, dedicatedworker, serviceworker). 
How do you convey the uuid from one context to the other?

Reply:
I guess they come from the uuid params of the URL?

Reply:
See https://github.com/web-platform-tests/rfcs/pull/88